index.html

```diff
@@ -9,17 +9,24 @@
   <title>Juan Diaz Portfolio</title>
 </head>
 
-<body class="body">
 
-  <section class="intro">
+
+<body class="body">
+  <div id="mobile-menu" class="mobile-menu">
+    <a class="m-navlink close-btn text-decoration-none" href="#">X</a>
+    <a class="m-navlink text-decoration-none" href="#portfolio">Portfolio</a>
+    <a class="m-navlink" href="#about">About</a>
+    <a class="m-navlink" href="#contactme">Contact</a>
+  </div>
+  <section id="intro" class="intro section">
     <header>
       <nav class="navbar">
         <a class="my-logo" href="#">My Logo</a>
         <div class="navlist">
           <a class="navlink" href="#portfolio">Portfolio</a>
           <a class="navlink" href="#about">About</a>
           <a class="navlink" href="#contactme">Contact</a>
-          <a class="menu-icon" href="#"></a>
+          <a id="hamburguer-btn" class="menu-icon" href="#"></a>
         </div>
       </nav>
     </header>
```

Comment:
- [ ] Your hamburger menu is not visible on the mobile version, kindly try to fix it.
- [ ] The social media icons are not displaying properly as you can see:

![rev1](https://user-images.githubusercontent.com/53347966/179715572-efa3c5b6-95eb-4f9a-8196-683458dadf6b.JPG)

Reply:
the icons issue must be on your side.